utils.go

```diff
@@ -40,3 +41,27 @@ func validateMountOptions(mountOptions []string) error {
 	}
 	return nil
 }
+
+func applyNameOperation(oldNames []string, names []string, op updateNameOperation) ([]string, error) {
+	result := make([]string, 0)
+	switch op {
+	case setNames:
+		// ignore all old names and just return new names
+		return names, nil
+	case removeNames:
+		// remove given names from old names
+		for _, name := range oldNames {
+			if !stringutils.InSlice(names, name) {
```

Comment:
I don’t think the case-folding behavior of `stringutils.InSlice` is expected here.

Reply:
using custom loop here, stringutils seems to be case insensitive, maybe later we can add a new function in stringutils itself.